src/daft-connect/src/spark_analyzer.rs

```diff
@@ -146,31 +146,28 @@ impl SparkAnalyzer<'_> {
             RelType::Deduplicate(rel) => self.deduplicate(*rel).await,
             RelType::Sort(rel) => self.sort(*rel).await,
             RelType::Sql(sql) => self.sql(sql).await,
-            plan => not_yet_implemented!("relation type: \"{}\"", rel_name(&plan))?,
+            plan => not_yet_implemented!("relation type: \"{}\"", rel_name(&plan)),
```

Comment:
Minor nit: Use raw string literals to avoid having to escape embedded double-quotes. E.g., something like:

```rs
not_yet_implemented!(r#"relation type: "{}""#, rel_name(&plan))
```

Just helps with readability, in my opinion.